test/win-loss.test.js

```diff
@@ -0,0 +1,92 @@
+import winLoss from '../src/win-loss.js';
+
+const test = QUnit.test;
+
```

Comment:
These tests all look great! Awesome job naming your tests to reflect which case you're testing for, and you did a good job testing for all possible cases. If you want to, you can use spaces in the name for your test (the way you have it is fine, just wanted to clarify in case you didn't know!)